# global_costmap:
#   global_frame: /map
#   robot_base_frame: base_link
#   update_frequency: 4.0
#   static_map: true


global_costmap:
  global_frame: /map
  robot_base_frame: base_link
  update_frequency: 5.0
  publish_frequency: 2.0
  static_map: true
  rolling_window: false
  width: 30.0
  height: 30.0
  resolution: 0.025
#   unknown_cost_value: 255

#map_topic: /map

# plugins: 
# - {name: simple_layer, type: "simple_layer_namespace::SimpleLayer"}

  # plugins: #[]
  #  - 
  #    name: static_map2
  #    type: "costmap_2d::StaticLayer"
  #    # map_topic: map2

  #  - 
  #    name: inflation_layer
  #    type: "costmap_2d::InflationLayer"

  
